jaeger-core/src/main/java/com/uber/jaeger/utils/RateLimiter.java

```diff
@@ -22,20 +22,20 @@
 package com.uber.jaeger.utils;
 
 public class RateLimiter {
-  private final double creditsPerSecond;
   private final double creditsPerNanosecond;
   private final Clock clock;
   private double balance;
+  private double maxBalance;
   private long lastTick;
 
-  public RateLimiter(double creditsPerSecond) {
-    this(creditsPerSecond, new SystemClock());
+  public RateLimiter(double creditsPerSecond, double maxBalance) {
+    this(creditsPerSecond, maxBalance, new SystemClock());
   }
 
-  public RateLimiter(double creditsPerSecond, Clock clock) {
+  public RateLimiter(double creditsPerSecond, double maxBalance, Clock clock) {
```

Comment:
I think it is unnecessary to force an user to compute `creditsPerSecond` and `maxBalance` to do the basic function of rate limiting.

As a glance, looking at something like `RateLimiter(creditsPerSecond=2, maxBalance=100)` doesn't immediately tell a reader that this is a RateLimiter that rate limits at around 100 requests per 50s when the item cost is 1.